core/admin/admin_routes.py

```
@@ -0,0 +1,580 @@
+        }
+    except Exception as e:
+        logger.error(f"Failed to scan ports: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+# =============================================================================
+# Export Configuration Endpoint
+# =============================================================================
+
+@monitoring_router.get("/export-prometheus-config")
+async def export_prometheus_config(request: Request):
```

Ruff (0.14.10)

Unused function argument: `request`

(ARG001)